src/components/manage/Sidebar/SidebarPopup.jsx

```diff
@@ -2,26 +2,43 @@ import React from 'react';
 import { Portal } from 'react-portal';
 import { CSSTransition } from 'react-transition-group';
 import PropTypes from 'prop-types';
-import { doesNodeContainClick } from 'semantic-ui-react/dist/commonjs/lib';
 
 const DEFAULT_TIMEOUT = 500;
 
 const SidebarPopup = (props) => {
   const { children, open, onClose, overlay } = props;
 
-  const asideElement = React.createRef();
-
   const handleClickOutside = (e) => {
-    if (asideElement && doesNodeContainClick(asideElement.current, e)) return;
-    onClose();
+    let sidebarContainer = e?.target?.closest('#sidebar-popup-container');
```

Comment:
I don't remember browser compatibility that we are granting with Volto, but please note that ``closest`` is not IE compatible. There's a [tiny polyfill at MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#polyfill), in case it's needed.

Reply:
IE is not supported: https://github.com/plone/volto/blob/2888ad2d927601cc1b1472086dd1cdd5d713994f/src/config/index.js#L107